src/models/slices/directanswer.ts

```diff
@@ -0,0 +1,5 @@
+import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '@yext/answers-core';
+
+export interface DirectAnswerState {
+  result?: FeaturedSnippetDirectAnswer | FieldValueDirectAnswer;
```

Comment:
I think result can be `undefined` as well for the types

Reply:
The question mark indicates that result may be undefined. Are you suggesting to make this  `FeaturedSnippetDirectAnswer | FieldValueDirectAnswer | undefined`?

Reply:
hm ok, I wasn't sure if the question mark would cover the case when we directly set it as undefined `state.result = action.payload;` compare to just it being undefined because it was never set. If that's the same thing, then all good!

Reply:
yep, they are the same